src/constants/lists.ts

```diff
@@ -17,14 +18,14 @@ const GEMINI_LIST = 'https://www.gemini.com/uniswap/manifest.json'
 const BA_LIST = 'https://raw.githubusercontent.com/The-Blockchain-Association/sec-notice-list/master/ba-sec-list.json'
 const HONEYSWAP_LIST = 'https://tokens.honeyswap.org'
 const LEVINSWAP_LIST =
-  'https://ipfs.io/ipfs/QmUWxthidUYXUJ2kiZLLPxkMKYDAinnpA591R3SRN6wufs?filename=levinswap-default.tokenlist.json'
+  'https://ipfs.io/ipfs/QmUmN7Be3LLHiEwcVZDm6WsPjcTddWsc6C7hrLCmPzsanv?filename=levinswap-default.tokenlist.json'
```

Comment:
this url changed?

Reply:
Yes! The old one was failing to fetch the list, seems that either the registry is not supported anymore in IPFS or they migrated it